src/phrases/phrases-nn.js

```diff
@@ -433,23 +433,25 @@ const phrases = {
     },
   },
   editorToolbar: {
-    bold: 'Feit',
-    italic: 'Kursiv',
-    underlined: 'Understreka',
-    quote: 'Sitat',
-    link: 'Lenke',
-    numberedList: 'Nummerert liste',
-    bulletedList: 'Punktliste',
-    twoColumnList: 'To-kolonners liste',
-    letterList: 'Bokstavliste',
-    headingOne: 'Overskrift 1',
-    headingTwo: 'Overskrift 2',
-    headingThree: 'Overskrift 3',
-    footnote: 'Fotnote',
-    mathml: 'Matte',
-    concept: 'Forklaring',
-    code: 'Kode',
+    bold: 'Feit\n(ctrl+b)',
+    bulletedList: 'Punktliste\n(ctrl+l)',
+    code: 'Kode\n(ctrl+k)',
     codeblock: 'Kodeblokk',
+    concept: 'Forklaring\n(ctrl+alt+c)',
+    footnote: 'Fotnote\n(ctrl+alt+f)',
+    headingOne: 'Overskrift 1',
+    headingThree: 'Overskrift 3\n(ctrl+3)',
+    headingTwo: 'Overskrift 2\n(ctrl+2)',
+    italic: 'Kursiv\n(ctrl+i)',
+    letterList: 'Bokstavliste\n(ctrl+alt+a)',
+    link: 'Lenke\n(ctrl+alt+l)',
+    mathml: 'Matte\n(ctrl+m)',
+    numberedList: 'Nummerert liste\n(ctrl+alt+1)',
+    quote: 'Sitat\n(ctrl+q)',
+    sub: 'Senka\n(ctrl+alt+s)',
+    sup: 'Hevet\n(ctrl+alt+h)',
+    twoColumnList: 'To-kolonners liste',
+    underlined: 'Understreka',
```

Comment:
er ikke `ctrl+u` vanlig for underline om det går an å legge til også?

Reply:
Jo, men vi har fjerna muligheten for underline på bestilling fra ndla for å ikkje blande det med lenker. Så hurtigtasten er borte.